.github/actions/report-job-status/action.yml

```diff
@@ -22,16 +16,15 @@ runs:
         # to the personal chat.
         if [[ ${{ github.ref }} == refs/heads/master ||
               ${{ github.ref }} =~ refs/heads/[0-9].[0-9]+$ ]]; then
-          echo CHAT_ID="${{ inputs.chat-id }}" | tee -a $GITHUB_ENV
+          echo CHAT_ID="tt_cicd_reports" | tee -a $GITHUB_ENV
```

Comment:
This can be defined as a env variable:

```
env:
  CHAT_ID: tt_cicd_reports
run: |
  # Report failures occurred in the 'master' and release branches to
  # the team chat. Report failures occurred in PRs and other branches
  # to the personal chat.
  if [[ ${{ github.ref }} == refs/heads/master ||
        ${{ github.ref }} =~ refs/heads/[0-9].[0-9]+$ ]]; then
    echo CHAT_ID=${{ env.CHAT_ID }} | tee -a $GITHUB_ENV
  else
    ..
  fi   
```

Reply:
fixed